### PR TITLE
feat add Title component

### PR DESCRIPTION
**Pull request by keremaydemir123**
Title component is added
Some css variables are added to globals.css file

### CODE REVIEW

File: components/Title/index.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import styles from "./Title.module.css";
+
+type TitleProps = {
+  level: number;
+  color?: string;
+  text: string;
+  className?: string;
+};
+
+const Title: React.FC<TitleProps> = ({ level, color, text, className }) => {
```
Comment (OrhanOzkercin) [inline components/Title/index.tsx:11]: I am not so good at React but do we need `React.FC` ?

**Comment by OrhanOzkercin** [comment]
Good job 👏 I have some comments. 

- You push package-lock and yarn.lock both, this is a problem.
- Also why do we need the AllTitles component, I did not understand that.

**Review by OrhanOzkercin — CHANGES_REQUESTED** [review]
Good job 👏 

You push package-lock and yarn.lock both, this is a problem.
Also why do we need the AllTitles component, I did not understand that.

**Comment by frontendship2** [comment]
Can you update this pr? Someone else already did this task. You just need to add styles on current Title.module.css file.

**Pull request closed without merging**